src/plugins/mediawiki.lua

```diff
@@ -7,31 +7,41 @@ local plugin = {}
 plugin.help = 'Usage: mediawiki <APIURL> <search>'
 
 plugin.main = function(args)
-    local _, _, apiurl, search = args.message:find('(%S+)%s+(.+)')
+    local _, _, apiurl, search = args.message:find('^[ ]?(https?://[^> ]+)%s+(.+)')
```

Comment:
Why have the leading `^[ ]`?

Additionally, `[^> ]` doesn't make sense since we aren't requiring (nor supporting) delimiting this URL with `<>`. So, the second character class should probably be `%S+`.

Reply:
Something like this? `^(https?://%S+)%s+(%S+)`

Reply:
that looks good, yeah.